README.md

```diff
@@ -21,7 +21,7 @@ Flytekit is the core extensible library to author Flyte workflows and tasks and
 ### Base Installation
 
 ```bash
-pip install flytekit==0.16.0b6
+pip install flytekit==0.16.0b7
```

Comment:
not relevant to this PR but it would be awesome if our automatic release process could also update versions in docs